test/test_automl/automl_utils.py

```diff
@@ -52,13 +60,50 @@ def print_debug_information(automl):
     return os.linesep.join(content)
 
 
+def _includes(scores, all_scores):
+    if len(all_scores) != len(scores):
+        return False
+
+    for score in all_scores:
+        if score not in scores:
+            return False
+
+    return True
+
+
 def count_succeses(cv_results):
     return np.sum(
         [status in ['Success', 'Success (but do not advance to higher budget)']
          for status in cv_results['status']]
     )
 
 
+def includes_all_scores(scores):
+    all_scores = scores_dict["train_single"] + scores_dict["test_single"] + \
+                 scores_dict["train_ensamble"] + scores_dict["test_ensamble"] + ["Timestamp"]
+    return _includes(scores, all_scores)
+
+
+def include_single_scores(scores):
+    all_scores = scores_dict["train_single"] + scores_dict["test_single"] + ["Timestamp"]
+    return _includes(scores, all_scores)
+
+
+def includes_train_scores(scores):
+    all_scores = scores_dict["train_single"] + scores_dict["train_ensamble"] + ["Timestamp"]
+    return _includes(scores, all_scores)
+
+
+def performance_over_time_is_plausible(poT):
+    if len(poT) < 1:
+        return False
+    if len(poT.drop(columns=["Timestamp"]).dropna()) < 1:
+        return False
```

Comment:
Are there `na` columns in the dataframe? From my understanding of the code this only happens if a `run` doesn't have an `endtime` which as far as I'm aware is always available in successful runs.

Reply:
yes you are right. This case shouldnt appear. At this point I'd say it doesnt hurt to have it in there. However, I'd be fine with removing it also. How should I proceed @eddiebergman

Reply:
It's fine as is, just something I was curious about! :)